tests/integration/tier0/test-check-rollback-handling/test_check_rollback_handling.py

```diff
@@ -87,19 +87,9 @@ def test_proper_rhsm_clean_up(shell, convert2rhel):
     Verify that usermode, rhn-setup and os-release packages are not removed.
     """
     install_pkg(shell)
-    if "oracle-7" in booted_os:
-        prompt_amount = 3
-    elif "centos-7" in booted_os:
-        # additional question about the removal of python-syspurpose
```

Comment:
We don't have this comment in the `main.fmf`. If you think it's not valuable anymore, then I'm ok with it. Otherwise, I guess we should add that there too?

Reply:
Oh, you're right.
I will add that there with rebase.
Thank you for noticing that @r0x0d !